marklogic-data-hub/src/main/resources/ml-modules/root/com.marklogic.hub/lib/job-lib.xqy

```diff
@@ -0,0 +1,82 @@
+xquery version "1.0-ml";
+
+module namespace job = "http://marklogic.com/data-hub/job-lib";
+
+import module namespace config = "http://marklogic.com/data-hub/config"
+  at "/com.marklogic.hub/lib/config.xqy";
+
+declare option xdmp:mapping "false";
+
+declare variable $SUCCESS-KEY := "success";
+declare variable $FAILED-KEY  := "failed";
+
+(:
+ : Given a list of job-ids, delete those jobs and any traces associated with them.
+ :
+ : ML 9.0-1.1 is the only DHF-supported version that does not support the <update> option
+ : for xdmp:invoke-function. (<transaction-mode> has been deprecated.)
+ :)
+declare function job:delete-jobs-and-traces($job-ids as xs:string*)
+{
+  let $job-results := job:delete-jobs($job-ids)
+  let $options :=
+    <options xmlns="xdmp:eval">
+      {
+        if (xdmp:version() = "9.0-1.1") then
```

Comment:
is 9.0-1.1 the only outlier here?

Reply:
among the supported versions (8.0-7+ and 9.0-1.1+), yes